Document/0x05a-Platform-Overview.md

```diff
@@ -540,6 +540,12 @@ With the APK signature scheme, the complete APK is hashed and signed, and an APK
 ![Preparation](Images/Chapters/0x05a/apk-validation-process.png)
 [APK signature verification process](https://source.android.com/security/apksigning/v2#verification "APK Signature verification process")
 
+#### APK Signature Scheme (v3 Scheme)
+
+The v3 APK Signing Block format is the same as v2. V3 adds information about the supported SDK versions and a proof-of-rotation struct to the APK signing block. In Android 9 and higher, APKs can be verified according to APK Signature Scheme v3, v2 or v1 scheme. Older platforms ignore v3 signatures and try to verify v2 then v1 signature.
+
+![apk-validation-process-v3-scheme](Images/Chapters/0x05a/apk-validation-process-v3-scheme)

```

Comment:
Can you add information about the actual signing key rotation and the best practices for this. And then add best practrices for it when you have to be backwards compatible as well...